examples/sldeditor/core/widgets/Handle.cpp

```diff
@@ -44,24 +48,38 @@ Handle::Handle(HandleItem* item) :
 void Handle::paint(QPainter* painter, const QStyleOptionGraphicsItem*, QWidget*)
 {
     painter->setBrush(color);
-    painter->drawEllipse(-10, -10, 20, 20);
+    painter->drawEllipse(
+        - handle_item->property(HandleItem::P_RADIUS).toDouble()/2,
+        - handle_item->property(HandleItem::P_RADIUS).toDouble()/2,
+        handle_item->property(HandleItem::P_RADIUS).toDouble(),
+        handle_item->property(HandleItem::P_RADIUS).toDouble()
+    );
```

Comment:
It looks like `QPainter::drawEllipse(int x, int y, int w, int h)` version of method is used here. I suggest to explicitly work with QPointF, QRectF in all painter's to avoid later problems with high-dpi.